integration-libs/s4om/components/order-detail-attachments/attachments-dialog/s4om-attachments-dialog.component.html

```diff
@@ -0,0 +1,169 @@
+                          attachment.attachmentId,
+                          attachment.fileName
+                        )
+                      "
+                      (keydown.enter)="
+                        openOrderAttachment(
+                          attachment.attachmentId,
+                          attachment.fileName
+                        )
+                      "
+                    >
+                      {{ attachment.fileName }}
+                    </a>
+                  </td>
+                  <td>
+                    <div
+                      class="inline-spinner"
+                      [ngClass]="{
+                        hidden: !loadingAttachments.includes(
+                          attachment.attachmentId
+                        ),
+                      }"
+                    >
+                      <cx-spinner></cx-spinner>
+                    </div>
+                  </td>
+                  <td>
+                    <a
+                      tabindex="0"
+                      role="button"
+                      class="file-icon"
+                      attr.aria-label="{{
+                        's4omOrder.attachments.dialog.preview' | cxTranslate
+                      }}"
+                      title="{{
+                        's4omOrder.attachments.dialog.preview' | cxTranslate
+                      }}"
+                      [ngClass]="{
+                        disabled: loadingAttachments.includes(
+                          attachment.attachmentId
+                        ),
+                      }"
+                      (click)="
+                        openOrderAttachment(
+                          attachment.attachmentId,
+                          attachment.fileName
+                        )
+                      "
+                      (keydown.enter)="
+                        openOrderAttachment(
+                          attachment.attachmentId,
+                          attachment.fileName
+                        )
+                      "
+                    >
+                      <cx-icon [type]="iconTypes.FILE"></cx-icon>
+                    </a>
+                  </td>
+                </tr>
+              }
+            } @empty {
+              <cx-message
+                [text]="'s4omOrder.attachments.dialog.empty' | cxTranslate"
+                [type]="globalMessageType.MSG_TYPE_INFO"
+                [isVisibleCloseButton]="false"
+                class="cx-dialog-message info-message"
+              >
+              </cx-message>
+            }
+          </table>
+        } @else {
+          <cx-message
+            [text]="'s4omOrder.attachments.dialog.error' | cxTranslate"
+            [type]="globalMessageType.MSG_TYPE_ERROR"
+            [isVisibleCloseButton]="false"
+            class="cx-dialog-message error-message"
+          >
+          </cx-message>
+        }
+      } @else {
+        <div class="cx-spinner">
```

Comment:
MAJOR: if attachments$ is not loaded, we are showing spinner. but we have attachmentsCount$ which is dependent on attachments$. Shouldn't this spinner be enclosing both attachments$ and attachmentsCount$